src/chef_index.app.src

```diff
@@ -18,12 +18,11 @@
 {application, chef_index,
  [
   {description, "Chef search index tools"},
-  {vsn, git},
+  {vsn, {cmd,"cat ../../VERSION | awk '{print $0}'"}},
```

Comment:
This bit assumes the chef-server's setup re the VERSION file, will that still make sense in the context it is being used?  Maybe we can change this to a static version number if it doesn't.